controllers/backplaneconfig_controller.go

```diff
@@ -696,12 +709,15 @@ func (r *MultiClusterEngineReconciler) ensureCustomResources(ctx context.Context
 	return ctrl.Result{}, nil
 }
 
-func (r *MultiClusterEngineReconciler) finalizeBackplaneConfig(ctx context.Context, backplaneConfig *backplanev1.MultiClusterEngine) error {
+func (r *MultiClusterEngineReconciler) finalizeBackplaneConfig(ctx context.Context, backplaneConfig *backplanev1.MultiClusterEngine, ocpConsole bool) error {
```

Comment:
We should leave out such a component-specific toggle in the method signature if possible